apps/webstore/src/app/shared/components/modals/add-to-basket-modal/add-to-basket-modal.component.html

```diff
@@ -124,6 +124,10 @@
                         <span class="horizontal">15 cm</span>
                         <span class="vertical">15 cm</span>
                     </span>
+                    <span *ngSwitchCase="'PET_SIGN'">
```

Comment:
is it possible to say 'QUOTABLE || PET_SIGN' or something like that? Since it uses the same sizes as quotable.